test/alignak_test.py

```diff
@@ -136,102 +137,92 @@ class AlignakTest(unittest.TestCase):
-            return
-        self.conf.clean()
-
-        self.confs = self.conf.cut_into_parts()
-        self.conf.prepare_for_sending()
-        self.conf.show_errors()
-        self.dispatcher = Dispatcher(self.conf, self.me)
-
-        scheddaemon = Alignak(None, False, False, False, None)
-        self.scheddaemon = scheddaemon
-        self.sched = scheddaemon.sched
-        scheddaemon.load_modules_manager()
-        # Remember to clean the logs we just created before launching tests
-        self.clear_logs()
-        m = MacroResolver()
-        m.init(self.conf)
-        self.sched.load_conf(self.conf)
-        e = ExternalCommandManager(self.conf, 'applyer')
-        self.sched.external_command = e
-        e.load_scheduler(self.sched)
-        e2 = ExternalCommandManager(self.conf, 'dispatcher')
-        e2.load_arbiter(self)
-        self.external_command_dispatcher = e2
-        self.sched.conf.accept_passive_unknown_check_results = False
-
-        self.sched.schedule()
+    @classmethod
+    def setUp(cls):
+        """
+        Load default configuration when start test
+
+        @verified
+
+        :return: None
+        """
+        cls.arbiter = Arbiter(['cfg/cfg_default.cfg'], False, False, False, False,
+                              '/tmp/arbiter.log')
+        cls.arbiter.load_config_file()
+        cls.scheduler = Alignak([], False, False, False, '/tmp/scheduler.log')
+        cls.scheduler.load_modules_manager()
+        for arb in cls.arbiter.conf.arbiters:
+            if arb.is_me():
+                cls.arbiter.myself = arb
+        cls.arbiter.dispatcher = Dispatcher(cls.arbiter.conf, cls.arbiter.myself)
+        for realm in cls.arbiter.conf.realms:
+            for sched in cls.arbiter.dispatcher.schedulers:
+                conf_package = {
+                    'conf': realm.serialized_confs.itervalues().next(),
+                    'override_conf': sched.get_override_configuration(),
+                    'modules': [],
+                    'satellites': {'pollers': [], 'reactionners': []},
+                    'instance_name': sched.scheduler_name,
+                    'push_flavor': random.randint(1, 1000000),
+                    'skip_initial_broks': sched.skip_initial_broks,
+                    'accept_passive_unknown_check_results':
+                        sched.accept_passive_unknown_check_results,
+                    'api_key': cls.arbiter.dispatcher.conf.api_key,
+                    'secret': cls.arbiter.dispatcher.conf.secret,
+                    'http_proxy': cls.arbiter.dispatcher.conf.http_proxy,
+                    'statsd_host': cls.arbiter.dispatcher.conf.statsd_host,
+                    'statsd_port': cls.arbiter.dispatcher.conf.statsd_port,
+                    'statsd_prefix': cls.arbiter.dispatcher.conf.statsd_prefix,
+                    'statsd_enabled': cls.arbiter.dispatcher.conf.statsd_enabled,
+                }
+                cls.scheduler.new_conf = conf_package
```

Comment:
You are overriding the same variable. Only the last loop will sork (I guess there is only one elem in loop)